samples/boards/nrf/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -276,7 +276,7 @@ static int gen_onoff_status(struct bt_mesh_model *model,
 	return 0;
 }
 
-/* Generic Level (lIGHTNESS) Server message handlers */
+/* Generic Level (lightness) Server message handlers */
```

Comment:
to be consistent with `/* Generic Level (TEMPERATURE) Server message handlers */` below:

```suggestion
/* Generic Level (LIGHTNESS) Server message handlers */
```